nixos/tests/nixos-rebuild-install-bootloader.nix

```
@@ -1,6 +1,10 @@
 import ./make-test-python.nix ({ pkgs, ... }: {
   name = "nixos-rebuild-install-bootloader";
 
+  # test builds separate 32GB image for vm
+  # on hydra fails with "Output limit exceeded"
+  meta.broken = true;
```

Wouldn't this be better?

```suggestion
  meta.hydraPlatforms = [ ];
```

And frankly I'm not sure we should be disabling this test and breaking bootloaders.

---

I thought about it, but then what would be the point of keeping it.
Just so ofborg can run it on PRs ? (I am not sure it can at the current state)
And it has never run successfully since creation in october 2023 on hydra.